stellarphot/tests/test_core.py

```diff
@@ -32,28 +32,44 @@ def test_camera_unitscheck():
     dark_current = 0.01 * u.electron / u.second
     pixel_scale = 0.563 * u.arcsec / u.pix
 
-    with pytest.raises(TypeError):
+    with pytest.raises(ValueError):
```

Comment:
I'm confused about why this passes. pydantic should have turned the `ValueError` into a `ValidationError`.  I'd suggest we check for that , and add some `match` language to make sure we are getting the expected failure:

```suggestion
    with pytest.raises(ValidationError,  match="gain"):
```

Reply:
Well, we are throwing `ValueError` in the Camera class if there are bad values, should I have been throwing `ValidationError`?

Reply:
OK, I changed the tests, and they still clear.